src/ecs-mcp-server/awslabs/ecs_mcp_server/modules/express.py

```diff
@@ -38,7 +39,15 @@
 def register_module(mcp: FastMCP) -> None:
     """Register Express Mode module tools with the MCP server."""
 
-    @mcp.tool(name="build_and_push_image_to_ecr")
+    @mcp.tool(
```

Comment:
LGTM - builds and pushes image to ECR - destructive and makes external API calls. No UTs were added.